block_producer_swap_bot/README.md

```diff
@@ -0,0 +1,66 @@
+# block-producer-swap-bot
+
+  * [Getting started](#getting-started)
+    * [Requirements](#getting-started-requirements)
+      * [Ubuntu 16.04 & 18.04](#ubuntu-1604--1804)
+  * [Block producers](#block-producers)
+    * [Configuration file](#configuration-file)
+    * [Usage](#usage)
+  * [Development](#development)
+  
+## Getting started
+
+<h3 id="getting-started-requirements">Requirements</h4>
+
+#### Ubuntu 16.04 & 18.04
+
+Go to swap-bot-directory:
+
+```bash
+$ cd block-producer-swap-bot
+```
+
+Install requirements with following commands:
+
+```bash
+$ pip3 install requirements.txt
+```
+
+## Block producers
+
+### Configuration file
+
+Open configuration file.
+
+```bash
+$ nano cli/config.ini
+```
+
+Paste into the config file following content:
+
+```
+[NODES]
+remnode=127.0.0.1:8888
+eth-provider=wss://ropsten.infura.io/ws/v3/<your infura id>
+[REM]
+block-producer-permission=<your account name>@active
+block-producer-private-key=<your private key for signing blocks>
+```
+
+Replace remnode, eth-provider, block-producer-permission, block-producer-private-key with your remnode host and port, a link to Ethereum node with websocket connection, your block producer's account and private key for signing blocks.
+
+
+### Usage
+
+To start approving swaps run the following command
+
+```bash
+$ python3 block_producer_swap_bot.py eth-swap-bot process-swaps
```

Comment:
lets create a common start script, so the operators run a single .sh file to run the node and the swap processors

Reply:
Done.